fe/src/app/services/animal.service.ts

```diff
@@ -0,0 +1,52 @@
+import { HttpClient } from '@angular/common/http';
+import { inject, Injectable, signal } from '@angular/core';
+import { Observable, tap } from 'rxjs';
+import { Animal } from '../models/animal.model';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class AnimalService {
+  private http = inject(HttpClient);
+
+  private readonly apiUrl = 'http://localhost:8000/api/animals/';
```

Comment:
Die URL sollten wir in einen ENV viel ziehen damit wir sie später entsprechend zentral ändern können